src/app/sessions/services/sessions-service.ts

```diff
@@ -28,13 +33,28 @@ export class SessionsService {
             .pipe(map(data => {return normalize(data, sessions)}));
     }
 
+    paginatedSearchSessions(searchCriterions: SearchCriteria[], requestOptions: PaginatedRequestOption): Observable<Page<SessionSearchResponse>> {
```

Comment:
Couldn't you always send pagination settings? You could also make use of HttpParams object which does the '?' and '&' magic for you. Also, why sending the sort direction after a colon? Is it some convention? I'd be more keen on sending just a path param like 'sortdir'